packages/components/components/elvis-toolbox/src/components/overlay/overlayStyles.ts

```diff
@@ -53,6 +53,11 @@ export const OverlayContainer = styled.div<{ fadeOut: boolean; noAnimation: bool
       display: grid;
       place-content: center;
       inset: 0;
+      pointer-events: none; // Allow clicks to pass through, to the backdrop
+
+      & > * {
+        pointer-events: all;
```

Comment:
Is it better to leave this as the default here instead?
```suggestion
        pointer-events: auto;
```

Reply:
`pointer-events: all` is apparently an SVG-thing? 🤔  https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events#syntax